hub.html

```diff
@@ -223,3 +236,5 @@ <h3 class="research-hub-sub-title">Loading models</h3>
 
 <script src="{{ site.baseurl }}/assets/hub-filter.js"></script>
 <script src="{{ site.baseurl }}/assets/hub-buttons.js"></script>
+<script src="{{ site.baseurl }}/assets/search-bar.js"></script>
```

Comment:
You can remove the search-bar script since you split the code into hub-search-bar.